<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1578052846853" stop="1578052896362" version="1.4.8">
    <name>All Test Suite</name>
    <title>All Test Suite : 平台管理员[]</title>
    <test-cases>
        <test-case start="1578052846874" stop="1578052846879" status="pending">
            <name>ReconciliationAgent2</name>
            <title>验证代理对账第二条数据</title>
            <description type="text">验证代理对账差别金额第二条数据</description>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=24&amp;version=1"/>
                <label name="story" value="代理对账"/>
                <label name="feature" value="财务管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="18336@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1578052846879" stop="1578052846879" status="pending">
            <name>ReconciliationAgent1</name>
            <title>验证代理对账差别金额数据第一条数据</title>
            <description type="text">验证代理对账差别金额第一条数据</description>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=23&amp;version=1"/>
                <label name="story" value="代理对账"/>
                <label name="feature" value="财务管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="18336@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1578052846880" stop="1578052846880" status="pending">
            <name>ReconciliationVip2</name>
            <title>验证对账差别金额第二条数据</title>
            <description type="text">验证会员对账差别金额第一条数据</description>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=21&amp;version=1"/>
                <label name="story" value="会员对账"/>
                <label name="feature" value="财务管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="18336@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1578052846880" stop="1578052846880" status="pending">
            <name>ReconciliationVip</name>
            <title>验证会员对账差别金额数据第一条数据</title>
            <description type="text">验证会员对账差别金额第一条数据</description>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=20&amp;version=1"/>
                <label name="story" value="会员对账"/>
                <label name="feature" value="财务管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="18336@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1578052846880" stop="1578052846880" status="pending">
            <name>ReconciliationVip3</name>
            <title>验证会员对账差别金额第三条数据</title>
            <description type="text">验证会员对账差别金额第三条数据</description>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=22&amp;version=1"/>
                <label name="story" value="会员对账"/>
                <label name="feature" value="财务管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="18336@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1578052865240" stop="1578052873261" status="failed">
            <name>onlineRecharge</name>
            <title>验证会员对账线上充值金额</title>
            <description type="text">验证会员对账线上充值金额</description>
            <failure>
                <message>AssertionError: expected [true] but found [false]</message>
                <stack-trace>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at complex.FinanceManagement.onlineRecharge(FinanceManagement.java:73)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps>
                <step start="1578052865370" stop="1578052872710" status="failed">
                    <name>onlineRecharge</name>
                    <title>1.进入财务管理-会员对账，2.获取线上充值金额，3.进入充值记录，4.查询今日在线充值</title>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=20&amp;version=1"/>
                <label name="story" value="会员对账"/>
                <label name="feature" value="财务管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="18336@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1578052873265" stop="1578052879637" status="failed">
            <name>upRecharge</name>
            <title>验证会员对账线下充值金额</title>
            <description type="text">验证会员对账线下充值金额</description>
            <failure>
                <message>AssertionError: expected [true] but found [false]</message>
                <stack-trace>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at complex.FinanceManagement.upRecharge(FinanceManagement.java:119)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps>
                <step start="1578052873269" stop="1578052879276" status="failed">
                    <name>upRecharge</name>
                    <title>1.进入财务管理-会员对账，2.获取线下充值金额，3.进入充值记录，4.查询今日后台充值和手动入款金额</title>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=20&amp;version=1"/>
                <label name="story" value="会员对账"/>
                <label name="feature" value="财务管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="18336@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1578052879640" stop="1578052883723" status="failed">
            <name>activityLoseWinLotty</name>
            <title>验证会员对账彩票实际输赢金额</title>
            <description type="text">验证会员对账实际输赢金额</description>
            <failure>
                <message>AssertionError: expected [true] but found [false]</message>
                <stack-trace>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at complex.FinanceManagement.activityLoseWinLotty(FinanceManagement.java:161)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps>
                <step start="1578052879643" stop="1578052883285" status="failed">
                    <name>activityLoseWinLotty</name>
                    <title>1.进入财务管理-会员对账，2.获取彩票实际输赢金额，3.进入报表管理-会员报表，4.获取今日实际输赢（包括退水金额）</title>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=20&amp;version=1"/>
                <label name="story" value="会员对账"/>
                <label name="feature" value="财务管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="18336@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1578052883725" stop="0" status="passed">
            <name>UnSettlement</name>
            <title>验证会员对账彩票未结算金额</title>
            <description type="text">验证会员对账彩票未结算金额</description>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=20&amp;version=1"/>
                <label name="story" value="会员对账"/>
                <label name="feature" value="财务管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="18336@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
